CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## [Unreleased][unreleased]
 
+## [0.11.1] - 2017/08/24
```

Comment:
2017/10/24

Reply:
Sorry, this is SF calendar where summertime just began.